enterprise/frontend/src/metabase-enterprise/whitelabel/components/HelpLinkSettings/HelpLinkRadio.tsx

```diff
@@ -0,0 +1,62 @@
+import { t } from "ttag";
+import { useCallback } from "react";
+import { Radio, Stack } from "metabase/ui";
+
+interface Props {
+  setting: { value: string };
+  onChange: (value: string) => void;
+}
+
+export const HelpLinkRadio = ({ setting, onChange }: Props) => {
+  const focusUrlInput = useCallback(async () => {
+    // The url input is rendered only after the setting is saved and returned from the BE
+    // there's no easy way to autofocus it from here besides waiting.
+    // `autofocus` can't be used otherwise it will be autofocused also on page load
+    const input = await waitFor(() =>
+      document.querySelector<HTMLInputElement>(
+        "#setting-help-link-custom-destination",
+      ),
+    );
+    input?.focus();
+  }, []);
+
+  return (
+    <Radio.Group value={setting.value} onChange={onChange}>
+      <Stack>
+        <Radio
+          label={t`Link to` + " Metabase " + t`help`}
```

Comment:
I'd leave this as 
```ts
t`Link to Metabase help`
```

as [Application name from appearance settings erroneously used for embedding license explanation](https://github.com/metabase/metabase/issues/34920#top) will be worked on soon possibly within this week. And I'm worried we'll forget to fix this afterward.

Or you could leave this as is but should there be a comment that this is a temporary bandage that should be fixed soon?